traffic_ops/testing/compare/compare.go

```diff
@@ -196,6 +197,32 @@ func readRespBodies(a *io.ReadCloser, b *io.ReadCloser) ([]byte, []byte, error)
 	return aBody, bBody, nil
 }
 
+// Scrubs out the traffic ops headers from the passed lines
+// Note that this assumes UNIX line endings
+func scrubPlainText(lines []string) string {
+	r := ""
+	for _, l := range lines {
+		if len(l) < len(SHORT_HEADER) {
+			r += l
+		} else if l[:len(SHORT_HEADER)] != SHORT_HEADER {
+
```

Comment:
I think this would be cleaner reversing this compare and `continue` if it matches..   same w/ LUA_HEADER below.  Reduces nested indents and makes it clear that there's nothing more to do if it matches